src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -154,11 +189,9 @@ private void driveArm(TrapezoidProfile.State state) {
         double armPIDVolts = armPID.calculate(getArmPos(), state.position);
         if ((getArmPos() > ARM_UPPER_LIMIT_RAD && state.velocity > 0) || 
             (getArmPos() < ARM_LOWER_LIMIT_RAD && state.velocity < 0)) {
-              forbFlag = true;  
+              forbFlag = true;
```

Comment:
If i recall correctly, "forbFlag" was used originally to "rumble" the controller but it never worked. I would not merge unworking code into master so let's get rid of it.